### PR TITLE
CORTX-31981: convert motor trace files from binary to yaml before bundling

### PR DESCRIPTION
**Pull request by rohit-k-dwivedi**
Signed-off-by: Rohit Dwivedi <rohit.k.dwivedi@seagate.com>
JIRA: https://jts.seagate.com/browse/CORTX-31981
# Problem Statement
- m0trace files are in binary format

# Design
-  converted m0trace files to yaml format using command `m0trace -i <input file> -Y -o <output file>`
# Coding
   Checklist for Author
-  [x] Coding conventions are followed and code is consistent

# Testing 
  Checklist for Author
- [ ] Unit and System Tests are added
- [ ] Test Cases cover Happy Path, Non-Happy Path and Scalability
- [x] Testing was performed with RPM

# Impact Analysis
  Checklist for Author/Reviewer/GateKeeper
- [ ] Interface change (if any) are documented
- [ ] Side effects on other features (deployment/upgrade)
- [ ] Dependencies on other component(s)

# Review Checklist 
  Checklist for Author
- [x] JIRA number/GitHub Issue added to PR
- [x] PR is self reviewed
- [x] Jira and state/status is updated and JIRA is updated with PR link
- [x] Check if the description is clear and explained

# Documentation
  Checklist for Author
- [ ] Changes done to WIKI / Confluence page / Quick Start Guide

### CODE REVIEW

File: src/rgw/support/rgw_support_bundle [hunk 1]
```diff
@@ -105,7 +105,8 @@ class RGWSupportBundle:
                 tmp_motr_trace_dir = os.path.join(RGWSupportBundle._tmp_src, 'motr_trace_files')
                 os.makedirs(tmp_motr_trace_dir, exist_ok=True)
                 outfile = os.path.join(tmp_motr_trace_dir, file)
-                shutil.copyfile(infile, outfile)
+                # Convert m0trace file to human readable yaml format
+                SimpleProcess(f'm0trace -i {infile} -Y -o {outfile}').run()
```
Comment (cdeshmukh) [inline src/rgw/support/rgw_support_bundle:109]: shouldn't outfile extension should be .yaml now given m0trace binary files been coverted to yaml format ?

Comment (rohit-k-dwivedi) [reply src/rgw/support/rgw_support_bundle]: done

**Review by suryakumar1024 — APPROVED** [review]

**Comment by rohit-k-dwivedi** [comment]
tested on local env:
```
[root@ssc-vm-rhev4-2781 motr_trace_files]# pwd
/root/deploy-scripts/k8_cortx_cloud/logs-cortx-cloud-2022-06-14_05-19/bundle-logs-cortx-server-ssc-vm-rhev4-2781-7b9b864bfd-w7s88-2022-06-14_05-19/bundle-logs-cortx-server-ssc-vm-rhev4-2781-7b9b864bfd-w7s88-2022-06-14_05-19/rgw/motr_trace_files
[root@ssc-vm-rhev4-2781 motr_trace_files]# ls -lh
total 78M
-rw-r--r-- 1 root root 51M Jun 14 05:25 m0trace.10.2022-06-14-06:42:06.yaml
-rw-r--r-- 1 root root 28M Jun 14 05:25 m0trace.403.2022-06-14-06:41:54.yaml
```

**Review by cdeshmukh — APPROVED** [review]

**Review by sachitanands — APPROVED** [review]
looks good

**Pull request merged**